synapse/models/inet.py

```diff
@@ -891,6 +893,16 @@ def getBaseModels():
                     ('logout', {'ptype': 'time', 'doc': 'The time the account logged out of the service.'})
                 ]),
 
+                ('inet:web:action', {'ptype': 'inet:web:action'}, [
+                    ('action', {'ptype': 'str:lwr', 'doc': 'The action performed'}),
+                    ('acct', {'ptype': 'inet:web:acct', 'doc': 'The web account associated with the action'}),
```

Comment:
@therealsilence - @vEpiphyte proposed that `acct` be required. Any reason why we shouldn't do that?

Reply:
Concur - sort of hard to say "an account did some action" without the account. I like the way we're trying to tighten up the model with various 'req' and 'ro' constraints where appropriate.

Reply:
`ro:1`

Reply:
fixed